lib/screen/name_input_screen.dart

```diff
@@ -29,44 +61,95 @@ class _UserNameScreenState extends State<UserNameScreen> {
+                      ),
+                      textAlign: TextAlign.center,
+                    ),
                   ),
-                ),
-              ),
-              Spacer(),
-              ElevatedButton(
-                onPressed: _submitName,
-                style: ElevatedButton.styleFrom(
-                  backgroundColor: Colors.greenAccent,
-                  minimumSize: Size(double.infinity, 60),
-                  textStyle: TextStyle(fontSize: 20),
-                ),
-                child: Text('Submit'),
+                  Center(
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      mainAxisSize: MainAxisSize.min,
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.only(bottom: 20),
+                          child: Text(
+                            'Enter your name:',
+                            style: TextStyle(
+                              fontSize: 18,
+                              fontWeight: FontWeight.bold,
+                              color: Colors.black,
+                            ),
+                          ),
+                        ),
+                        Focus(
+                          onFocusChange: (hasFocus) {
+                            setState(() {
+                            });
+                          },
+                          child: TextField(
+                            controller: _nameController,
+                            decoration: InputDecoration(
+                              enabledBorder: OutlineInputBorder(
+                                borderSide: BorderSide(
+                                  color: Colors.grey, // Default border color
+                                  width: 2.0,
+                                ),
+                                borderRadius: BorderRadius.circular(12.0),
+                              ),
+                              focusedBorder: OutlineInputBorder(
+                                borderSide: BorderSide(
+                                  color: Color(0xFF098EAB), // Border color when focused
+                                  width: 2.0,
+                                ),
+                                borderRadius: BorderRadius.circular(12.0),
+                              ),
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  ScaleTransition(
+                    scale: _isButtonPressed ? _buttonScaleAnimation : AlwaysStoppedAnimation(1),
+                    child: ElevatedButton(
+                      onPressed: _submitName,
+                      style: ElevatedButton.styleFrom(
+                        backgroundColor: Color(0xFF098EAB),
+                        minimumSize: Size(double.infinity, 50),
```

Comment:
I don't see a necessity of adding minimum size here; you achieve the same by using Padding